polygeist/tools/cgeist/Test/Verification/gettimeofday.c

```diff
@@ -8,19 +8,19 @@ double alloc() {
   return Tp.tv_sec + Tp.tv_usec * 1.0e-6;
 }
 
-// clang-format off
 // CHECK:   func @alloc() -> f64
 // CHECK-NEXT:     %cst = arith.constant 9.9999999999999995E-7 : f64
-// CHECK-NEXT:     %alloca = memref.alloca() : memref<1x2xi64>
-// CHECK-NEXT:     %0 = "polygeist.memref2pointer"(%alloca) : (memref<1x2xi64>) -> !llvm.ptr<struct<(i64, i64)>>
-// CHECK-NEXT:     %1 = llvm.mlir.null : !llvm.ptr<i8>
+// CHECK-NEXT:     %c1_i64 = arith.constant 1 : i64
+// CHECK-NEXT:     %0 = llvm.alloca %c1_i64 x !llvm.struct<(i64, i64)> : (i64) -> !llvm.ptr<struct<(i64, i64)>>
+// CHECK-NEXT:     %1 = llvm.mlir.null : !llvm.ptr<struct<(i32, i32)>>
```

Comment:
on the buildbot machine, this line expects to be `%1 = llvm.mlir.null : !llvm.ptr<i8>`
```suggestion
// CHECK-NEXT:     %1 = llvm.mlir.null : !llvm.ptr<{{.*}}>
```